service/src/main/java/io/camunda/service/license/LicenseType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.license;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LicenseType {
+  SAAS("saas"),
+  PRODUCTION("production"),
+  UNKNOWN("unknown");
```

Comment:
so does unknown actually mean nothing has been provided? Like it is unlicensed?

Reply:
Correct, `unknown` is the error state. No license has been provided, the license is invalid, or an error occured when processing the license